mne/time_frequency/psd.py

```diff
@@ -95,7 +105,7 @@ def psd_array_welch(x, sfreq, fmin=0, fmax=np.inf, n_fft=256, n_overlap=0,
 
     Returns
     -------
-    psds : ndarray, shape (..., n_freqs) or
+    psds : ndarray, shape (..., n_freqs) or (..., n_windows, ..., n_freqs)
```

Comment:
you can't have more than one ellipsis, so `(..., n_windows, :, n_freqs)`.

Also I don't quite understand this. By the docs `x` can be 1D. What is the return size, then? Maybe it should be the -2 instead of -3 dim, i.e., `(..., n_windows, n_freqs)` then it's fine.